### PR TITLE
Feature/GitHub workflow

### PR DESCRIPTION
**Pull request by Benimanela**

### CODE REVIEW

File: github/github-workflow.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        <Parameter name="perPage" label="Per Page" required="false" default="30" />
+        <Parameter name="include" label="Include" required="false" default="web" />
+        <Parameter name="timeZone" label="Time Zone" required="false" default="UTC" />
+    </Parameters>
+
+    <Actions>
+        <!-- Clear the log source status before a new workflow run starts -->
+        <ClearStatus />
+
+        <!-- Initialize the risingValue filter for the first run. -->
```
Comment (sarbqdev) [inline github/github-workflow.xml:15]: add a description of how long we fetch in the first time in this value <Initialize path="/bookmark" value="${time() - 3600000}" /> also, maybe can we extract it as a parameter for the user? and set it 1 hr by def.

File: github/github-workflow.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<Workflow name="Qmasters GitHub workflow for QRadar" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V2">
+    <Parameters>
+        <Parameter name="orgName" label="Organization Name" required="false" />
+        <Parameter name="apiToken" label="API Token" required="true" secret="true" />
+        <Parameter name="phrase" label="Search Phrase" required="false"/>
+        <Parameter name="perPage" label="Per Page" required="false" default="30" />
```
Comment (sarbqdev) [inline github/github-workflow.xml:6]: add def for 'perPage' to 100 it is not big and why not? also in parameters file.

**Review by sarbqdev — COMMENTED** [review]
please fix the comment, and also support pagination.

File: github/github-workflow.xml (hunk 1 above)
Comment (Benimanela) [reply github/github-workflow.xml:15]: description added and added startTime parameter to use for the first run.

File: github/github-workflow.xml (hunk 2 above)
Comment (Benimanela) [reply github/github-workflow.xml:6]: changed to 100

**Review by LiorQM — APPROVED** [review]

File: github/github-workflow.xml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                <CallEndpoint url="${/nextLink}" method="GET" savePath="/getLogs">
+                    <BearerAuthentication token="${/apiToken}" />
+                </CallEndpoint>
+
+                <!-- Handle Errors -->
+                <If condition="/getLogs/status_code != 200">
+                    <Log type="ERROR" message="GitHub API error: ${/getLogs}"/>
+                    <Abort reason="GitHub abort reason: ${/getLogs/body/message}" />
+                </If>
+
```
[the remark this answers is not in the document]
Comment (Benimanela) [reply github/github-workflow.xml]: fixed

File: github/github-workflow.xml [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        </If>
+
+        <!-- Post Events, if any -->
+        <If condition="count(/getLogs/body) > 0">
+            <PostEvents path="/getLogs/body" source="${/identifier}" />
+        </If>
+
+        <!-- Prepare the lastEventsTime path -->
+        <Set path="/lastEventsTime" value="[]"/>
+        <Set path="/eventsIDs" value="[]"/>
```
[the remark this answers is not in the document]
Comment (Benimanela) [reply github/github-workflow.xml]: fixed the workflow counters

**Pull request merged**